src/test/unittest/configurator.py

```diff
@@ -277,6 +277,9 @@ def ctx_choices(cls):
                             'e.g.: 0-2,5 will execute TEST0, '
                             'TEST1, TEST2 and TEST5',
                             default='')
+        parser.add_argument('--list-tests', dest='list_tests', help='list'
+                            ' the tests including configurations with which'
```

Comment:
Don't break the lines like this, move the help field entirely to the second line, so that you don't have a single word up there. At first, I didn't understand that this is also part of the help text.

This doesn't read well, and doesn't explain well what it does. `Prints all (or only those that pass filters?) tests, including context parameters, to the standard output.` Something like that (but not exactly this...).